package.json

```diff
@@ -25,7 +25,8 @@
     "d3": "^5.15.1",
     "next": "^9.3.4",
     "react": "^16.13.1",
-    "react-dom": "^16.13.1"
+    "react-dom": "^16.13.1",
+    "rxjs": "^6.5.5"
```

Comment:
I would rather remove this dependency right now we don't want to add complexity and keep the weight of the app to load fast

Reply:
I agree, i will remove it.